ext/nnstreamer/tensor_decoder/tensordec-boundingbox.c

```diff
@@ -2210,6 +2210,72 @@ void
 init_bb (void)
 {
   nnstreamer_decoder_probe (&boundingBox);
+
+  {
+    g_autofree gchar *sub_desc = g_strjoinv ("|", (GStrv) bb_modes);
+    const gchar *desc =
```

Comment:
const? possible mem leak.

Reply:
Although init_bb is supposed to be called only once, I'd suggest to have a static variable and let it be gfree-ed if called multiple times. Then, you may choose to gfree it at fini_bb() if you've decided to have a global-static.

Reply:
> const? possible mem leak.

Is it freed by https://github.com/nnstreamer/nnstreamer/blob/e7710dff2b94ef809f4794367240d5d4f13e5306/gst/nnstreamer/nnstreamer_subplugin.c#L54 at the time when ```unregister_subplugin``` is called?